NimbleTests/Matchers/SatisfyAnyOfTest.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+import Nimble
+
+class SatisfyAnyOfTest: XCTestCase {
+    func testSatisfyAnyOf() {
```

Comment:
We need one more case: If no matchers a given to `satisfyAnyOf`, then we should fail with an error like:

```
satisfyAnyOf requires at least one matcher
```